src/_content/posts/djangocon-us-grants.md

```diff
@@ -0,0 +1,63 @@
+---
+author: Communications Team
+category: General
+published_datetime: 2025-04-29 11:00:00
+title: "DjangoCon US Grants"
+---
+DjangoCon US isn't just a conference.. People come for the code and stay for the community. It's a conference for Pythonistas, web developers, open-source developers, and anyone who would like to learn something new about Django. It's where people come to know, connect, and grow. 
+
+Everyone passionate about Django should have the chance to participate in this experience. That's why we have the [Opportunity Grants program for DjangoCon US 2025](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform), and we wholeheartedly encourage you to [apply](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform).
+
+
+Opportunity Grants can be as **simple as a free ticket**, whether you’re attending in person or online.
+
+
+Whether you are a Django professional who wants to share your expertise, a student just beginning your journey with web development, or simply someone curious about the framework, these grants are designed to make DjangoCon US accessible. You do not need to be a speaker or a first-time attendee to be eligible. If you're reading this and considering applying, then yes, you should [apply](https://docs.google.com/forms/d/e/1FAIpQLSfuriCI4JtAEcTfLy9Qdk2h_DfmaGKE544svRfMJLCOu5eLDQ/viewform)!
+
+
+Important Dates to Remember:
+- Application Deadline: **May 4, 2025**, at Noon Eastern Daylight Time (America/New_York).
+- Decision Notifications: Will be sent out by **June 23, 2025**.
+- Recipients of Opportunity Grants will receive separate instructions on how to upload necessary receipts and provide their payment information to facilitate reimbursement.
+- Reimbursements are made a week after the conference, which typically lasts 4 to 5 weeks, to consolidate. Be patient, as the team is working tirelessly to get everyone sorted. In case of questions during that period, reach out to us via [email](mailto:grants@djangocon.us)
```

Comment:
```suggestion
- Reimbursements are made **a week after the conference**, which typically lasts 4 to 5 weeks. Please be patient during that time, as the team is working tirelessly to get everyone sorted. In case of questions, reach out to us via [email](mailto:grants@djangocon.us)
```